CIScripts/Test/PesterLogger/PesterLogger.Tests.ps1

```diff
@@ -68,13 +82,58 @@ Describe "PesterLogger" -Tags CI, Unit {
             Initialize-PesterLogger -OutDir "TestDrive:\"
             { Write-Log "msg1" } | Should -Throw
         }
+
+        Context "Timestamps" {
+            $RealGetDate = Get-Command Get-Date -CommandType Cmdlet
+            Mock Get-Date {
+                & $RealGetDate `
+                    -Year 2018 `
+                    -Month 6 `
+                    -Day 8 `
+                    -Hour 14 `
+                    -Minute 46 `
+                    -Second 44 `
+                    -Millisecond 42 `
+                    @Args
+            }
+
+            It "are present in logs" {
+                Initialize-PesterLogger -OutDir "TestDrive:\"
+                Write-Log "foo"
+                Get-Content "TestDrive:\PesterLogger.Write-Log.Timestamps.are present in logs.txt" |
+                    Should -Be "2018-06-08 14:46:44.042000 | test-runner | foo"
+            }
+
+            It "can be disabled" {
+                Initialize-PesterLogger -OutDir "TestDrive:\"
+                Write-Log -NoTimestamps "foo"
+                Get-Content "TestDrive:\PesterLogger.Write-Log.Timestamps.can be disabled.txt" |
+                    Should -Be "                           | test-runner | foo"
```

Comment:
We should probably use `ConvertTo-LogItem` here and in `It "are present in logs"` as well

Reply:
I think it's good to have a test that checks the exact format and I didn't want to create a separate one just for that.

Reply:
I disagree, just like you shouldn't test if your UI shows a red button or a green button. Formatting is meant to be changed.

Or at least have at most literally one test case for exact formatting method. But I expect it to change a lot and the test is fragile.